docs/configuration.adoc

```diff
@@ -34,16 +36,16 @@ are marked as important ones.
 
 This strategy can be configured in several ways through Java system properties:
 
-* `scm.range.head` and `scm.range.tail` are used to set the range of commits you want to inspect for new tests.
+* `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="COMMIT"]` and `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="LAST_COMMITS"]` are used to set the range of commits you want to inspect for new tests.
 The values can be commit ids or using `HEAD` notation.
-For example: `-Dscm.range.head=HEAD -Dscm.range.tail=HEAD~`
+For example: `-Dconst:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="COMMIT"]=HEAD -Dconst:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="LAST_COMMITS"]=HEAD~`
 By default if not specified, default value is `HEAD`.
-* `scm.last.changes` can be used to set the number of commits from `HEAD` that you want to inspect.
-For example `-Dscm.last.changes=3` will be equivalent as `-Dscm.range.head=HEAD -Dscm.range.tail=HEAD~~`.
+* `const:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="LAST_COMMITS"]` can be used to set the number of commits from `HEAD` that you want to inspect.
+For example `-Dconst:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="LAST_COMMITS"]=3` will be equivalent as `-Dconst:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="COMMIT"]=HEAD -Dconst:core/src/main/java/org/arquillian/smart/testing/scm/ScmRunnerProperties.java[name="LAST_COMMITS"]=HEAD~~`.
```

Comment:
Here is the same mistake. 
Maybe, it would be worth changing the variable names to something that reflects the content. eg:
```java
    public static final String SCM_RANGE_TAIL = "scm.range.tail";
    public static final String SCM_RANGE_HEAD = "scm.range.head";
    public static final String SCM_LAST_CHANGES = "scm.last.changes";
```
instead of:
```java
    public static final String PREVIOUS_COMMIT = "scm.range.tail";
    public static final String COMMIT = "scm.range.head";
    public static final String LAST_COMMITS = "scm.last.changes";
```

Reply:
I have not checked yet but you mean that the value is correct, what happened is that you get confused because of the variables names right?

Reply:
Should we rename the variable names to this PR? I think it should be another one, since this was just a PR for updating docs, I think this improvement (that should be done) should be a separate issue.

Reply:
it's more than that: https://github.com/arquillian/smart-testing/pull/150#discussion_r139092538

Reply:
YEs fixing it right now, but the rename task I think it should be another PR

Reply:
yeah, can be :-)